plugins/dashboard/views/status.html

```diff
@@ -0,0 +1,116 @@
+<html>
+<title>Dashboard</title>
+<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0-beta/css/bootstrap.min.css" integrity="sha384-/Y6pD6FV/Vv2HJnA6t+vslU6fwYXjCFtcEpHbNJ0lyAFsXTsjBbfaDjzALeQsN6M" crossorigin="anonymous">
+<link href="https://fonts.googleapis.com/css?family=Open+Sans:400,700" rel="stylesheet">
+<link type="text/css" rel="stylesheet" href="/css/dashboard.css" media="all">
+<body>
+<div class="row no-gutters">
+  <div class="col">
+    <div class="timer--wrapper">
+      <div class="timer" id="timer">
+      </div>
+    </div>
+  </div>
+</div>
+<div class="row no-gutters header--inner align-items-center">
+  <div class="col-3">
+    <img src="/img/dashboard_logo.svg" class="logo">
+  </div>
+  <div class="col-6">
+    <h1 class="header">{{ page_title }}</h1>
+  </div>
+  <div class="col-3">
+    <div class="calendar--wrapper">
+      <div class="calendar">
+        <div class="month">{{ date.month}}</div>
+        <div class="date">{{ date.day }}</div>
+        <div class="year">{{ date.year }}</div>
+      </div>
+    </div>
+  </div>
+</div>
+
+{% if not screens.critical is empty %}
+  <div class="row no-gutters critical">
+    <table class="table text-danger">
+      <col style="width:5%">
+      <col style="width:18%">
+      <col style="width:10%">
+      <col style="width:67%">
+      <thead>
+      <tr>
+        <th>Id</th>
+        <th>Last beat</th>
+        <th>Installation</th>
+        <th>Screen</th>
+      </tr>
+      </thead>
+      <tbody>
+        {% for data in screens.critical %}
+          {% for screen in data %}
+            <tr>
+              <td>{{ screen.id }}</td>
+              <td>{{ screen.heartbeat }}</td>
+              <td>{{ screen.name }}</td>
+              <td>{{ screen.title }}</td>
+            </tr>
+          {% endfor %}
+        {% endfor %}
+      </tbody>
+    </table>
+  </div>
+{% endif %}
+
+<div class="footer">
+  <div class="row no-gutters footer--inner">
+    <div class="col-8 col-offset-2 mx-auto summed-updates">
+      <table class="table text-info">
+        <thead>
+        <tr>
+          <th># of screen</th>
+          <th>Critical</th>
+          <th>Black listed</th>
+        </tr>
+        </thead>
+        <tbody>
+          <td>{{ screens.count.total }}</td>
+          <td>{{ screens.count.critical }}</td>
+          <td>{{ screens.count.blacklist }}</td>
+        </tbody>
+      </table>
+    </div>
+  </div>
+</div>
+<script>
```

Comment:
Inline script?

Reply:
Yes, alternative it has to be placed in the /public folder